test/e2e/utils.go

```diff
@@ -50,6 +54,16 @@ func writePodsLogs(namespace string, writer io.Writer) error {
 	return nil
 }
 
+func interfacesName(interfaces []interface{}) []string {
+	var names []string
+	for _, iface := range interfaces {
+		name, hasName := iface.(map[string]interface{})["name"]
+		Expect(hasName).To(BeTrue(), "should have name field in the interfaces, nhttps://github.com/nmstate/nmstate/blob/master/libnmstate/schemas/operational-state.yam")
```

Comment:
there is trailing "n" before the link, missing l at the end

Reply:
Done

Reply:
I will try to fix in same commit a do smaller PRs.